017-FrontDoor/Coach/Solution00.md

```diff
@@ -0,0 +1,20 @@
+### Notes for Challenge 0 - Setup your Environment and Discover
```

Comment:
All coach solution .md files should use H1 (#) for their titles like the student guide does.

Reply:
done